src/extra/gpkg/wkb.jl

```diff
@@ -0,0 +1,163 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    GeoIO.gpkgwkbgeom(io, crs, wkbtype, zextent, wkbbyteswap);
+
+Flavors of WKB supported:
+
+- Standard WKB supports two-dimensional geometry, and is a proper subset of both extended WKB and ISO WKB.
+
+## Reading WKB Geometry BLOB
+
+``` julia
+    io = IOBuffer(WKBGeometryBLOB)
+    # load byte order in order to transport geometry 
+    # easily between systems of different endianness
+    wkbByteOrder = read(io, UInt8)
+    # load simple feature wkb geometry types supported
+    wkbGeometryType = read(io, UInt32)
+    # wkb points do not have a `numPoints` field 
+    if wkbGeometryType != 1
+      # load number of geometries in geometry set
+      numWkbGeometries = read(io, UInt32)
+    end
+    # load in WKBGeometry that contain geometry values
+    # w/ double precision numbers in the coordinates
+    # that are also subject to byte order rules.
+    wkbEndianness = isone(wkbByteOrder) ? ltoh : ntoh
+    # Note that Julia does not convert the endianness for you.
+    wkbGeometryBlob = wkbEndianness(read(io, Vector{UInt8}))
+```
```

Comment:
A docstring is written with a single purpose: to teach readers how a function is supposed to be used, nothing more. Please move or erase these design details to inside the implementation for those interested in learning the code.

The current implementation of gpkgwkbgeom is so simple, and yet the docstring is massive (and disconnected) with the implementation.